modules/gitops-release-notes-1-12-2.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-12-2_{context}"]
+= Release notes for {gitops-title} 1.12.2
+
+{gitops-title} 1.12.2 is now available on {OCP} 4.12, 4.13, 4.14, and 4.15.
+
+[id="errata-updates-1-12.2_{context}"]
+== Errata updates
+
+[id="RHSA-2024:2816-gitops-1-12-2-security-update-advisory_{context}"]
+=== RHSA-2024:2816 - {gitops-title} 1.12.2 security update advisory
+
+Issued: 2024-05-10
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-2024:2816[RHSA-2024:2816]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="new-features-1-12-2_{context}"]
+== New features
+
+* With this update, support has been provided for the `must-gather` tool in the Argo Rollouts Operator. This update provides support for the following enhancements:
```

Comment:
@Dhruv-Soni11 Please add an attribute for Argo Rollouts Operator just like how we have for the GitOps Operatorand include it in the common-attributes.adoc file. Apply this attribute all thru the doc for all occurrences of Argo Rollouts Operator.

This is Global comment, applying to all versions.

Reply:
Fixed.